components/api/db/queries.bal

```diff
@@ -20,6 +20,8 @@ public const string GET_ORG_QUERY = "SELECT DESCRIPTION, WEBSITE_URL, FIRST_AUTH
                                     FROM REGISTRY_ORGANIZATION WHERE ORG_NAME = ?";
 public const string ADD_ORG_QUERY = "INSERT INTO REGISTRY_ORGANIZATION ( ORG_NAME, DESCRIPTION, WEBSITE_URL,
                                     DEFAULT_IMAGE_VISIBILITY, FIRST_AUTHOR ) VALUES ( ?,?,?,?,? )";
+public const string ADD_ORG_USER_MAPPING_QUERY = "INSERT INTO REGISTRY_ORG_USER_MAPPING ( USER_UUID, ORG_NAME, USER_ROLE) 
+                                                  VALUES ( ?,?, \"admin\")";
```

Comment:
Let's parameterize the admin as well. We will need a more generic one in the long run